Evergreen/Manifests/IrfanView.json

```diff
@@ -0,0 +1,45 @@
+﻿{
+    "Name": "IrfanView",
+    "Source": "https://www.irfanview.com/",
+    "Get": {
+        "Uri": "https://www.irfanview.com/",
```

Comment:
I need to update docs on the manifest structure, but this should be structured like the example here: https://github.com/aaronparker/evergreen/blob/2a54df73a5cc25aa0b65064dfed3d19b38a887fe/template/Template.json